spec/models/converters/date_time_converter_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe DateTimeConverter do
+  subject { described_class }
```

Comment:
this is unused.  I think conventionally the tests should be using `subject` instead of repeating `described_class` below

Reply:
I can remove line 6 and just use `described_class`, but in order to use `subject` explicitly we'd have to disable the rubocop `RSpec/NamedSubject` rule or it'd throw a linter error.  If you think that's a better solution I can make a separate PR to update `.rubocop.yml`, but I'd rather not have this wait on that if possible.